Makefile.work

```diff
@@ -43,6 +43,7 @@
 #  * ENABLE_AUTO_TECH_SUPPORT: Enable the configuration for event-driven techsupport & coredump mgmt feature
 #  *                 Default: y
 #  *                 Values: y,n
+#  * ENABLE_YANG_CONFIG_VALIDATION: Enable YANG validation of updates to ConfigDB
```

Comment:
>ENABLE_YANG_CONFIG_VALIDATION [](http://example.com/codeflow?start=5&length=29)

This name is too general, considering we already use yang model to validate some config.

Suggest `ENABLE_CLI_YANG_VALIDATION`

Reply:
I realize the `CLI` is not a proper name either. Let's keep this discussion open for a while.